src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -167,13 +156,7 @@ public static LocalizableResourceString GetDescription(ErrorCode code)
 
         public static string GetHelpLink(ErrorCode code)
         {
-            string helpLink;
-            if (s_helpLinksMap.Value.TryGetValue(code, out helpLink))
-            {
-                return helpLink;
-            }
-
-            return string.Empty;
+            return $"https://msdn.microsoft.com/query/dev16.query?appId=Dev16IDEF1&l={CultureInfo.CurrentCulture}&k=k({GetId(code)});k(DevLang-csharp)&rd=true.";
```

Comment:
This is the same way NuGet works:

https://github.com/NuGet/NuGet.Client/blob/4277fadddd8b17c551f57c9688abbd34d604f61f/src/NuGet.Clients/NuGet.VisualStudio.Common/ErrorListTableEntry.cs#L17

and also the same way Visual Studio currently uses for keywords help.

Reply:
Currently the C# error messages documentation is scattered in two directories:

https://github.com/dotnet/docs/tree/master/docs/csharp/language-reference/compiler-messages
https://github.com/dotnet/docs/tree/master/docs/csharp/misc

After that is fixed, we can use:

```csharp
return $"https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-messages/{GetId(code)}";
// example: https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-messages/CS1001
```